### ISSUE TITLE
Get green screen when maximizing after invoking scrcpy

### ISSUE THREAD

**Issue by RndyP** [Genymobile/scrcpy#2112]
Running scrcpy 1.17 on Windows 7 SP1. Android device is Google Pixel 3. I am running an MFC app that is controlling the Camera app on the pixel 3. After I invoke scrcpy I want to maximize the main window. I use SendInput() to send keystrokes and mouse events. After invoking scrcpy, if the first thing I do is send Windows Logo key with up arrow (this works if done with physical keyboard) I get a solid green screen. My hack to get it to work is to send a Windows Logo key with down arrow (minimize) before maximize. This works around the problem. I tried inserting a Sleep(4000) before maximize, thinking it's timing related, but that didn't help.

**Comment by RndyP** [comment]
Also get the green screen with a Samsung S2. Seems like it's not device related. I've discovered that if I start the app with SW_SHOW instead of SW_MAXIMIZE in ShellExecuteEx(), and then later, maximize using fake keystrokes, don't get the green. If I had to guess, it's some kind of timing related thing with the Windows messaging.

**Comment by rom1v** [comment]
Does this also occur on older scrcpy versions?

**Comment by RndyP** [comment]
@rom1v Yes, does it with 1.10. Easy to reproduce. All you have to do is create a shortcut to scrcpy on the Windows 10 desktop, right click the shortcut, then change the properties to start Maximized. Hook up your android device, double click the shortcut, and viola! Green. Surprised not too many users are seeing this. Maybe others not starting maximized??? I'd ask the developers if maybe thread timing? Initialization sequences?

**Comment by RndyP** [comment]
@rom1v In regards to Anthony's "tablet mode" green screen problem. I switched my Windows 10 box into tablet mode, and invoked scrcpy from the command line, and guess what? Green screen! If you are in "tablet mode" scrcpy is maximized by default running from a Command prompt, unlike "non-tablet mode". So the problem is simply first running scrcpy maximized. In tablet mode Windows does this by default.

**Comment by rom1v** [comment]
I can reproduce on Windows in a VM. I'll investigate later.

But note that you could use `scrcpy -f` (in your shortcut or when you run from a terminal) to start in fullscreen instead of maximized (and there is no green screen).

**Comment by RndyP** [comment]
@rom1v The reason I don't use scrcpy -f is because of issue #2118
